civis/io/_tables.py

```diff
@@ -1,5 +1,5 @@
 import csv
-import tempfile
+import io
```

Comment:
Actually, what is the purpose of the try/except around that import? `from io import StringIO` works in 2.7 and 3.5.

http://python-future.org/compatible_idioms.html#stringio